apps/contracts/src/contracts/SlmEscrow.sol

```diff
@@ -64,14 +64,14 @@ contract SlmEscrow is SlmShared {
     }
 
     /// Allow either party to withdraw if eligible
-    function withdrawFunds() external {
-        require(msg.sender == _party1 || msg.sender == _party2, 'Only parties can withdraw');
-        require(judge.voteStatus(address(this)) == 3, 'Cannot withdraw');
+    function withdrawFunds(bytes32 encryptionKey) external {
+        require(msg.sender == _party1 || msg.sender == _party2, "Only parties can withdraw");
+        require(judge.getVoteResults(address(this), encryptionKey) == 2 || judge.getVoteResults(address(this), encryptionKey) == 3, "Cannot withdraw");
```

Comment:
Line is long and a little confusing, maybe calculations `judge.getVoteResults(address(this), encryptionKey) == 2` (and `== 3`) to a separate variable, or even helper functions, e.g. `party1CanWithdraw` or similar.

Reply:
Separated out to clarify conditions